frontend/coprs_frontend/coprs/logic/coprs_logic.py

```diff
@@ -285,6 +285,7 @@ def get_multiple_fulltext(cls, fulltext=None, projectname=None,
         return query
 
     @classmethod
+    @no_autoflush
```

Comment:
This disables just one warning :shrug:

Reply:
I think we can drop the `no_autoflush` entirely in favor of https://github.com/fedora-copr/copr/pull/3699